GDJS/Runtime/events-tools/windowtools.js

```diff
@@ -22,6 +22,13 @@ gdjs.evtTools.window.setFullScreen = function(runtimeScene, enable, keepAspectRa
     runtimeScene.getGame().getRenderer().setFullScreen(enable);
 };
 
+gdjs.evtTools.window.isFullScreen = function() {
+	if (typeof require === "function") {
+		return require("electron").remote.getCurrentWindow().isFullScreen();
+	}
+	return window.screen.height === window.innerHeight;
```

Comment:
Could you move the implementation into the renderer (`gdjs.RuntimeGamePixiRenderer`)? 
The idea is to avoid having any HTML5 specific code in these events, specifically everything that is using `window`. 

In the future, we may have a new renderer which does not run in a browser :) (for example, something that runs in Node.js only, for a server only game, or something like Cocos2D-JS which embeds a JS engine but not a browser).

Reply:
Then inside, the pixi renderer, uses `var electron = this.getElectron(); if (electron) { ... } else { ... }`, so that it is consistent with the rest of the Electron specific code :) 

Note that there might be other Electron specific code in the game engine/extensions (like the file system), but in this case as it's code in `windowtools.js` related to the window size, we put everything in the renderer.